### ISSUE TITLE
Cannot customize lambda function name

### ISSUE THREAD

**Issue by andreas-eberle** [quarkusio/quarkus#16188]
## Describe the bug
We have a project with multiple lambda functions. As described in the documentation, it is easily possible to select the lambda function we want Quarkus to run (https://quarkus.io/guides/amazon-lambda#choose). Unfortunately, it is not possible to upload multiple lambda functions to different names because the `FUNCTION_NAME` is hardcoded in the `manage.sh` script and cannot be customized. Furthermore, instead of deriving the `FUNCTION_NAME` from the configured `quarkus.lambda.handler`, the artifact name is used (https://github.com/quarkusio/quarkus/blob/main/extensions/amazon-lambda/common-deployment/src/main/java/io/quarkus/amazon/lambda/deployment/LambdaUtil.java#L81).

Therefore, it is currently not viable to have multiple lambda functions in the same module as you cannot upload them to different lambda functions in AWS. 

### Expected behavior
The lambda function name should be derived from the `quarkus.lambda.handler` property.

### Actual behavior
The artefact name is used. 

## Environment (please complete the following information):

Quarkus 1.13.0.Final

**Comment by patriot1burke** [comment]
Isn't this just a matter of modifying manage.sh?

**Comment by andreas-eberle** [comment]
Yes, I think it is only a matter of modifying the manage.sh template. However, I didn't manage to find out how to inject the function name into there.
